pfcpiface/pfcpiface.go

```diff
@@ -29,7 +29,7 @@ type sessRecord struct {
 
 var sessions map[uint64]sessRecord
 
-func pfcpifaceMainLoop(upf *upf, n3ip string, sourceIP string) {
+func pfcpifaceMainLoop(upf *upf, accessip string, sourceIP string) {
```

Comment:
`accessIP`?

Reply:
other places too. Also do we have to advertise our `coreIP` as well during association setup so that SMF knows for N9 interfacing?

Reply:
Let's talk about the N9 interface soon.